### PR TITLE
Creating basic files

### PR DESCRIPTION
**Pull request by BregornOriginal**
In this project I create a repository to practice the best way to make a good commit, a good branch and to create a pull request to be reviewed by another coding partner

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,17 @@
...
+
+- Git & Github Overflow
+- Linters for HTML, CSS and JS
+- Create Branch whit best practices
+- Best practices of commiting
+- Best practices for Pull Request
+
+Live Demo (Work on progress).
+
+![Esta es una imagen](https://camo.githubusercontent.com/38f113b96a368dfb7f634d2f2da97e7b8c748042d2a284b97c3fad048bb3ff55/68747470733a2f2f6d69726f2e6d656469756d2e636f6d2f6d61782f323733322f312a6d74736b3366515f4252656d466964686b656c3364412e706e67)
```
Comment (phelian23) [inline README.md]: - [ ] Good job adding a README.md file to your project, however, it's not descriptive enough. Please use [this template](https://github.com/microverseinc/readme-template) but remember to customize it to your project.

**Review by phelian23 — CHANGES_REQUESTED** [review]
Hi @BregornOriginal, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

- [x] Kindly add linter files, and make sure they are all passing. Follow the instructions here [ setting up a linter for HTML and CSS](https://github.com/microverseinc/linters-config) to set it up.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (BregornOriginal) [reply README.md]: Hi @phelian23, thank you for reviewing my code! Ok, I'm trying to get working linters, but I'm stuck. This is the error that I'm getting.
image

 I'm using Prettier, I'ts not useful as linters?. I have updated the readme whit more info.

Comment (BregornOriginal) [reply README.md]: @phelian23, I was not adding the correct files, that's why the linters were not working for me.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,49 @@
...
+
+- Images, Media and Backgrounds
+
+- Daily standup
+
+Contact me at:
+
+- GitHub: [BregornOriginal](https://github.com/BregornOriginal).
+
+![Esta es una imagen](https://camo.githubusercontent.com/38f113b96a368dfb7f634d2f2da97e7b8c748042d2a284b97c3fad048bb3ff55/68747470733a2f2f6d69726f2e6d656469756d2e636f6d2f6d61782f323733322f312a6d74736b3366515f4252656d466964686b656c3364412e706e67)
```
Comment (zainsadaqat) [inline README.md]: Hi, @BregornOriginal!
As requested by the previous Code Reviewer, Please use this [template](https://github.com/microverseinc/readme-template) provided by Microverse 😄

**Review by zainsadaqat — CHANGES_REQUESTED** [review]
# Hello @BregornOriginal  😃 😄 

You're doing a great job on your project. We're proud of you. Good job so far! 
# 👏🏻 👏🏻 👏🏻 

There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
  ✅  Correct use of Github Flow
  ✅  No linter errors
  ✅  Descriptive Pull Request

## Required Changes ♻️

_Check the comments under the review._




## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding! 👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 2 above)
Comment (BregornOriginal) [reply README.md]: Hi, @zainsadaqat! Thanks for your answer. I really don't know what mean whit the template. I'm not seeing the mistake. Can you give a hint?

Comment (BregornOriginal) [reply README.md]: @zainsadaqat It's ok now?

Comment (BregornOriginal) [reply README.md]: Hello @zainsadaqat , sorry if I'm bothering you, but this is my second request review. So I will be so glad if you can check the readme before I do another requested review. I don't want to repeat this for just a readme. Thank you!

Comment (zainsadaqat) [reply README.md]: ## Hey, @BregornOriginal! 😃 
Thanks for working on the requirements. You've successfully completed all the requirements for this project. You're good to go now. 🎉 
Please submit your PR for a review!

**Review by JuliCarracedo — APPROVED** [review]
# Status: Approved✔️

Hi @BregornOriginal , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**